clippy_lints/src/casts/mod.rs

```diff
@@ -754,6 +755,32 @@ declare_clippy_lint! {
     "detects `as *mut _` and `as *const _` conversion"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    /// Checks for casts of a primitive method pointer to any integer type.
```

Comment:
This communicates that casting *any* method on any integer type will cause this lint to fire, even though it checks for only min and max.

Reply:
Good point, clarified it.